Modules/Backup-VCSA/Backup-VCSA.psm1

```diff
@@ -1,204 +1,462 @@
+		This function will remove any scheduled backups of your
+		VCSA appliance. (vSphere 6.7 and higher)
+	.DESCRIPTION
+		Use this function to remove the backup schedule for your VCSA appliance.
+	.EXAMPLE
+		PS C:\> Remove-VCSASchedule
+	.EXAMPLE
+		PS C:\> Remove-VCSASchedule -ScheduleID 1
+	.NOTES
+		Credit goes to @AlanRenouf & @vBrianGraf for sharing the base of this function.
+		You must be connected to the CisService for this to work, if you are not connected, the function will prompt you for your credentials
+#>
+	[CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
+	param (
+		$ScheduleID = "default"
+	)
+	Begin {
+		if (!($global:DefaultCisServers)){ 
+			Add-Type -Assembly System.Windows.Forms
+			[System.Windows.Forms.MessageBox]::Show("It appears you have not created a connection to the CisServer. You will now be prompted to enter your vCenter credentials to continue" , "Connect to CisServer") | out-null
+			$Connection = Connect-CisServer $global:DefaultVIServer 
+		} else {
+			$Connection = $global:DefaultCisServers
+		}
+	}
+	Process{
+		if ((get-cisservice).name -notcontains "com.vmware.appliance.recovery.backup.schedules" ) {
+			Write-Error "This VCSA does not support Backup Schedules."
+			return
+		}
+		if ($PSCmdlet.ShouldProcess($ScheduleID, "Removes Current Backup Schedule")) {
+			$BackupAPI = Get-CisService -name 'com.vmware.appliance.recovery.backup.schedules'
+			$BackupAPI.delete($ScheduleID)
+		}
+	}
+	End {}
+}
 
-    End {}
+Function Test-VCSAScheduleSupport {
+<#
+	.NOTES
+	===========================================================================
+	 Original Created by:  Brian Graf
+	 Blog:                 www.vtagion.com
+	 Twitter:              @vBrianGraf
+	 Organization:         VMware
+	 Created / Modifed by: Michael Dunsdon
+	 Twitter:              @MJDunsdon
+	 Date:                 September 16, 2020
+	===========================================================================
+	.SYNOPSIS
+		This function will check to see if your VCSA supports Scheduled Backups.
+		(vSphere 6.7 and higher)
+	.DESCRIPTION
+		Use this function to check if your VCSA supports Scheduled Backups.
+	.EXAMPLE
+		PS C:\> Test-VCSAScheduleSupport
+	.NOTES
+		Credit goes to @AlanRenouf & @vBrianGraf for sharing the base of this function.
+		You must be connected to the CisService for this to work, if you are not connected, the function will prompt you for your credentia
+#>
+	param ()
+	Begin {
+		if (!($global:DefaultCisServers)){ 
+			Add-Type -Assembly System.Windows.Forms
+			[System.Windows.Forms.MessageBox]::Show("It appears you have not created a connection to the CisServer. You will now be prompted to enter your vCenter credentials to continue" , "Connect to CisServer") | out-null
+			$Connection = Connect-CisServer $global:DefaultVIServer 
+		} else {
+			$Connection = $global:DefaultCisServers
+		}
+	}
+	Process{
+		if ((get-cisservice).name -contains "com.vmware.appliance.recovery.backup.schedules" ) {
```

Comment:
nit: this works but to be more PowerShell you can use

(Get-CisServer -Name "com.vmware.appliance.recovery.backup.schedules" -ErrorAction SilentlyContinue) -eq $null

This could make performance improvement some day when server takes care of -Name filtering.